src/sql/workbench/contrib/notebook/browser/find/notebookFindModel.ts

```diff
@@ -2,6 +2,7 @@
  *  Copyright (c) Microsoft Corporation. All rights reserved.
  *  Licensed under the Source EULA. See License.txt in the project root for license information.
  *--------------------------------------------------------------------------------------------*/
+'use strict';
```

Comment:
This is not necessary. https://github.com/microsoft/azuredatastudio/blob/main/extensions/shared.tsconfig.json#L9 will ensure this is added automatically.

Reply:
reverted